src/main.rs

```diff
@@ -31,10 +31,14 @@ mod server;
 
 #[derive(Parser, Debug)]
 #[clap(author, version, about)]
-#[clap(group(ArgGroup::new("jwt").required(true).multiple(false).args(&["jwt_token", "jwt_path"])))]
+#[clap(group(ArgGroup::new("jwt").multiple(false).args(&["jwt_token", "jwt_path"])))]
 struct Args {
     /// JWT token for authentication
-    #[arg(long, env)]
+    #[arg(
+        long,
+        env,
+        default_value = "688f5d737bad920bdfb2fc2f488d6b6209eebda1dae949a8de91398d932c517a"
```

Comment:
I don't think there should be a default value here as the jwt token varies widely across deployments. This value is already in the .env.example file

Reply:
I did not know that you could use env for this too. I will remove this one then.